src/components/banner/Banner.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Icon from '../icon';
+import { BannerType, BannerIcon } from './BannerUtils';
+
+type BannerProps = {
+  /** Optional CSS class name */
+  className?: string;
+  /** If true, close icon will be placed to the right */
+  canClose?: boolean;
```

Comment:
As we have a prop in Dropdown that is "isClearable", I would go for "isClosable", but I don't have a strong opinion

Reply:
Sure! Better to have something homogeneous. I will change it. 👍